packages/contracts/src/framework/plugin/repo/PluginRepo.sol

```diff
@@ -71,9 +68,8 @@ contract PluginRepo is
     /// @param versionHash The tag hash.
     error VersionHashDoesNotExist(bytes32 versionHash);
 
-    /// @notice Thrown if a contract does not inherit from `PluginSetup`.
-    /// @param invalidPluginSetup The address of the contract missing the `PluginSetup` interface.
-    error InvalidPluginSetupInterface(address invalidPluginSetup);
```

Comment:
What is the reasoning behind removing it? Errors should be verbose and give you all the required information.
Imagine you are installing multiple plugins, and you want to find out which one was the wrong plugin setup.